events/cognito.go

```diff
@@ -266,3 +266,24 @@ type CognitoEventUserPoolsCustomMessageResponse struct {
 	EmailMessage string `json:"emailMessage"`
 	EmailSubject string `json:"emailSubject"`
 }
+
+// CognitoEventUserPoolsCustomEmailSender is sent by AWS Cognito User Pools before each mail to send.
+type CognitoEventUserPoolsCustomEmailSender struct {
+	CognitoEventUserPoolsHeader
+	Request  CognitoEventUserPoolsCustomEmailSenderRequest  `json:"request"`
+	Response CognitoEventUserPoolsCustomEmailSenderResponse `json:"response"`
```

Comment:
The docs don't mention a returned response, I think the sender integration isn't quite the same as the Custom Message integration

Email Sender: https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-custom-email-sender.html
> The custom code of your function must then process and deliver the message.

Compare with Custom Message Trigger: https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-custom-message.html#cognito-user-pools-lambda-trigger-syntax-custom-message
> In the response, specify the custom text to use in messages to your users.